src/base/rss/private/rss_parser.cpp

```diff
@@ -29,15 +29,16 @@
  * Contact : chris@qbittorrent.org
  */
 
+#include "rss_parser.h"
```

Comment:
To separate an additional prefix from the base name of the file (which usually matches the name of the class). Should I remind you why we use these additional prefixes for files in subdirectories of the project? Since we can have the same file names in different subfolders, it can lead to conflict when you build using qmake (because it builds everything in one folder).